Makefile

```diff
@@ -72,6 +75,9 @@ info: ## Show environment info.
 cargo-license:
 	FILES="$$(find . -name Cargo.toml | xargs grep -L '^license')"; if [ -n "$$FILES" ]; then echo "Missing license in: $${FILES}"; false; fi
 
+check-dirty-doc:
+	GEN="$$(./devtools/doc/rpc.py rpc/json/rpc.json)"; CUR="$$(cat rpc/README.md)"; if [ "$$GEN" != "$$CUR" ]; then echo "Dirty RPC document, please check"; false; fi
```

Comment:
Just generate the file into target file, and check whether it is dirty via `git diff --exit-code rpc/README.md`.